.ci/integration_test

```diff
@@ -248,10 +272,38 @@ function run_test_on_cluster() {
   fi
 }
 
-if [ "$1" == "cluster" ]; then
+function run_test_on_tm() {
+  if [ "$ACCESS_KEY_ID" == "" ] || [ "$SECRET_ACCESS_KEY" == "" ] || [ "$REGION" == "" ] ; then
+    echo "AWS S3 credentials unavailable. Exiting."
+    exit 1
+  fi
+
+  # TODO: remove
+  echo "Access ID: ${ACCESS_KEY_ID}, Access key: ${SECRET_ACCESS_KEY}, Region: ${REGION}"
```

Comment:
Remove this.

Reply:
I just added for testing something. Not to worry, I've removed it.